ush/yaml_from_parm.sh

```diff
@@ -5,11 +5,11 @@
 if [[ "$1" == "jedivar" ]]; then
   sed -e "s/@analysisDate@/${analysisDate}/" -e "s/@beginDate@/${beginDate}/" \
       -e "s/@HYB_WGT_STATIC@/${HYB_WGT_STATIC}/" -e "s/@HYB_WGT_ENS@/${HYB_WGT_ENS}/" \
-      "${EXPDIR}/config/jedivar.yaml" > jedivar.yaml
+      "${PARMrrfs}/jedivar.yaml" > jedivar.yaml
```

Comment:
@hu5970  We intentionally use `${EXPDIR}/config/` here.
see PR https://github.com/NOAA-EMC/rrfs-workflow/pull/745

Reply:
changed